raiden/ui/cli.py

```diff
@@ -204,6 +205,13 @@ def options(func):
             help="Show all configuration values used to control Raiden's behavior",
             is_flag=True,
         ),
+        option(
+            "--blockchain-sync-interval",
```

Comment:
Can you rename this option to "blockcain-query-interval" here and everywhere else where this name is used?

Since this is not about syncing the blockchain (aka downloading blocks until we reach the tip) but instead it's about querying the ethereum client.